setup.py

```diff
@@ -49,7 +49,10 @@ def read(fname):
         'boto >= 2.32',
         'bz2file',
         'requests',
-        'boto3'
+        # Temporary pin boto3 & botocore, because moto doesn't work with new version
+        # See https://github.com/spulec/moto/issues/1793
+        'boto3 < 1.8.0',
```

Comment:
Isn't moto needed only for testing? `install_requires` affects all users… should we restrict them from using newer boto3/botocore, just because of our testing dependency?

Reply:
Moto needed only for testing, that's true, I understand that this influences to all users, but this is "temporary" only (no big deal if we slightly limit boto version for some time).

As alternative solution, I can revert this boto* restriction and instead of it, install pinned versions manually (for travis only), this can be better solution, wdyt @piskvorky @mpenkov?

Reply:
Up to you; of course not limiting users is preferable, but it's probably no big deal, like you say.

Reply:
If we can’t run our unit tests with the newest boto3, then can we be sure
things really work with that boto3 version?

If the answer is no, then we should restrict our users to boto3 versions that we know work.

Reply:
This is breaking things downstream from you, 17 days later. 

Please revert this, or move to a development-only dependency, or else it will be necessary for us to stop using this library.

Reply:
@petergaultney hi, can you please describe, why does it break something? What's a problem with it?

Reply:
yeah, #227 is my exact issue!  thanks!